packages/vitest/src/node/cli.ts

```diff
@@ -48,6 +48,7 @@ cli
   .option('--inspect-brk', 'Enable Node.js inspector with break')
   .option('--test-timeout <time>', 'Default timeout of a test in milliseconds (default: 5000)')
   .option('--bail <number>', 'Stop test execution when given number of tests have failed', { default: 0 })
+  .option('--retry <number>', 'Retry the test specific number of times if it fails.', { default: 1 })
```

Comment:
This should also be mentioned in CLI documentation

Reply:
done, see change in `docs/guide/cli.md`

I also found some other CLI options which not appear in the documentation:
- single-thread
- hideSkippedTests
- segfaultRetry
- test-timeout